src/cmd/destroy.ts

```diff
@@ -32,22 +34,30 @@ const destroyAll = async () => {
     debugStream,
   )
   debug.info('Uninstalling all charts...')
-  await hf({ args: 'destroy' }, { streams: debugStream })
+  await stream(
+    nothrow($`helm ls -a -d --all-namespaces | awk 'NR > 1 { print "-n "$2, $1}' | xargs -L1 helm delete`),
```

Comment:
I believe that idea behind using JS is to leverage its power of parsing, yet I still see parsing SHELL scripts that are quite difficult to understand. 
Would it be possible to parse json output?
E.g.: 
```
data = JSON.parse($`helm ls -a -d --all-namespaces -o json`)
data.foreach( ... )
```